cardano-cli/test/data/golden/shelley/genesis/verification.key

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "GenesisVerificationKey_ed25519",
+    "description": "Genesis Verification Key",
```

Comment:
Same here. These already exist in: `/home/jordan/Repos/Work/cardano-haskell/cardano-node/cardano-cli/test/data/golden/shelley/keys/genesis_keys/`